docs/api/browser-window.md

```diff
@@ -869,6 +869,12 @@ the supplied bounds.
 
 Returns [`Rectangle`](structures/rectangle.md)
 
+#### `win.getBormalBounds()`
```

Comment:
tiny typo here
`getBormalBounds()` => `getNormalBounds()`

Reply:
Thanks for the typo. It realized some features are not well supported in CI and skipped in automatic tests (maximize, minimize, fullscreen). This is not always consistent, sometimes according to CI mode, sometimes according to OS. I will have a look at.